ui/app/pages/swaps/view-quote/view-quote.js

```diff
@@ -570,6 +571,7 @@ export default function ViewQuote() {
       </div>
       <SwapsFooter
         onSubmit={() => {
+          setSubmitClicked(true)
```

Comment:
I noticed that this never gets set back to `false` if the `signAndSendTransactions` throws an error. I artificially introduced an error and tested it out, and it just hangs with the button disabled.

We should instead show an error message and let the user retry. Or if we recognize the error as indication that the swap will almost certainly fail, then show an error and abort the swaps flow.

I think this can be dealt with separately from this PR though. This flow didn't work very well before this PR either - I don't think it's appreciably worse now.